src/drift/Arguments/ArgumentOptions.cs

```diff
@@ -0,0 +1,75 @@
+﻿using CommandLine;
+using System;
+using System.IO;
+
+namespace rangers.antidrift.drift.Arguments
```

Comment:
I would suggest using CamelCase for namespace and variables.

Reply:
You are completely right. I haven't noticed that it was somehow all lower case. I'll fix this in a bit.

Reply:
Resolved in the latest commit.